app/channels/game_channel.rb

```diff
@@ -133,16 +133,19 @@ def attack
       Handcard.draw_handcards(@gameboard.current_player.id, @gameboard, current_player_treasure)
       # TODO: add helping player to gameboard? give treasures to helping player
       if @gameboard.helping_player
-        helping_player = @gameboard.helping_player
-        Handcard.draw_handcards(helping_player, @gameboard, shared_reward)
+        helping_player_id = @gameboard.helping_player
+        helping_player = Player.find(helping_player_id)
+        Handcard.draw_handcards(helping_player_id, @gameboard, shared_reward)
+
+        PlayerChannel.broadcast_to(helping_player.user, { type: 'HANDCARD_UPDATE', params: { handcards: Gameboard.render_cards_array(helping_player.handcard.ingamedecks) } })
```

Comment:
i hab gwusst wir haben des nu irgendwo gehabt mit da player id speichern, hab voi vergessen dass des da helping player is.... is dawei schätz i ok aber vl sollt ma des auch auf des player object ändern